### DISCUSSION TITLE
Allow folder exclusion / backup selection

### DISCUSSION THREAD

**Discussion by alex27riva** [mrrfv/open-android-backup#60] [General]
Hi, I backup up my phone (Pixel 6a with stock Android 13), but there are some folder in the internal storage which are very heavy and I would like to exclude, is this currently possible?
Some examples:
- `Android/data/de.danoeh.antennapod/` this folder contains downloaded podcast that are useless to backup. The app database can be exported from the app settings.
- `Android/data/org.telegram.messenger/` this folder contains Telegram data / cache useless to backup because Telegram data is saved  on the cloud
- `Android/data/app.organicmaps/` downloaded maps, no need to backup.

The backup could be completed in less time by excluding some folders and avoiding copying quanta of unnecessary data.

**Comment by mrrfv** [comment]
Not possible at the moment, but I can add this feature once I have enough free time (in a few months at worst).
I think the best way to implement this would be to add an exclusions menu in the Android companion app, which would then tell the script about the folders to exclude. This approach makes persistence easy to implement, so there is no need to add the same exclusions every time the script is run. What do you think?

**Reply by gaborbit** [reply]
I would be happy with an exclusions config file, we could add useless folders and recommendations by default and everyone could add there preference also. GUI is nice but not as important from my perspective, I prefer editing this list on PC. (I would skip companion app's saving contact and sms option also if it would be possible)

update:
Would this work if I delete folders from the backup? I have not tried restore yet
